settings.json

```diff
@@ -2,6 +2,7 @@
   "accessibility.signals.terminalBell": {
     "sound": "off"
   },
+  "chat.disableAIFeatures": true,
```

Comment:
@rongxin-liu Is this necessary if it's in cs50.dev now already?

Reply:
Okay, going to omit, only so that these settings are overrides from cs50.dev itself.